docs/frameworks/nextjs.md

```diff
@@ -0,0 +1,29 @@
+# Next.js
+
+Next.js is a React framework that allows you to build static and server-rendered web applications. It is a popular choice for developers who want to build modern web applications with React.
+
+## Deployment
+
+To deploy a Next.js application with Genezio, all you need to do is to run the following command in the root directory of your project:
+
+```bash
+genezio deploy
+```
+
+This command will build your Next.js application and deploy it to the cloud. In case you don't have a `genezio.yaml` configuration file in your project, Genezio will create one for you by asking a few questions.
```

Comment:
In the most cases, the users will not have a `genezio.yaml` so I think its best here to mention that `genezio deploy` will create a file called `genezio.yaml` that will contain configs for the project.

Define a small version of the yaml here with comments that briefly explain:
```
# The name of your project, which also influence the subdomain of the project.
name: genezio-project
# Select a region closest to your user base for optimal performance. Available regions are:...
region: us-east-1
# Specifies the version of the YAML configuration syntax being used.
```

I would not necessarily point to the full documentation page because there are a lot of fields there that do not make sense for Nextjs.